shared/src/sms/sms.schema.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from "zod";
+import { Flavor } from "../typeFlavors";
+import { TemplatedSms } from "./smsTemplateByName";
+
+export type Phone = Flavor<string, "Phone">;
```

Comment:
```suggestion
export type SmsRecipient = Flavor<string, "SmsRecipient">;
```
Attention ici c'est pas simple de comprendre niveau namming qu'on veut un téléphone.
Et c'est à déplacer dans `shared/src/sms/smsTemplateByName.ts`

Reply:
tu as raison, je vais renommer en Phone partout

Reply:
ah attend, tu dis que c'est pas simple de comprendre qu'on veut un téléphone mais tu suggères un naming différent ? Je ne comprends plus trop

Reply:
bon j'ai gardé le type phone et j'ai renommé les variables : `recipientPhone` un peu partout